app/controllers/CheckYourAnswersController.scala

```diff
@@ -124,59 +97,164 @@ class CheckYourAnswersController @Inject()(
+    SummaryListViewModel(
+      rows = Seq(
+        BusinessNameSummary.row(request.registration),
+        BusinessVRNSummary.row(request.registration),
+        ReturnPeriodSummary.row(request.userAnswers)
+      ).flatten
+    ).withCssClass("govuk-!-margin-bottom-9")
+  }
+
   def onSubmit(period: Period): Action[AnyContent] = cc.authAndGetData(period).async {
     implicit request =>
       val vatReturnRequest =
         vatReturnService.fromUserAnswers(request.userAnswers, request.vrn, period, request.registration)
 
-      vatReturnRequest match {
-        case Valid(returnRequest) =>
-          vatReturnConnector.submit(returnRequest).flatMap {
-            case Right(vatReturn) =>
-
-              auditService.audit(ReturnsAuditModel.build(
-                returnRequest, SubmissionResult.Success, Some(vatReturn.reference), Some(vatReturn.paymentReference), request
-              ))
-
-              auditService.audit(ReturnForDataEntryAuditModel(returnRequest, vatReturn.reference, vatReturn.paymentReference))
-
-              emailService.sendConfirmationEmail(
-                request.registration.contactDetails.fullName,
-                request.registration.registeredCompanyName,
-                request.registration.contactDetails.emailAddress,
-                service.getTotalVatOwedAfterCorrections(request.userAnswers),
-                period
-              ) flatMap {
-                emailConfirmationResult =>
-                  val emailSent = EMAIL_ACCEPTED == emailConfirmationResult
-
-                  for {
-                    updatedAnswers <- Future.fromTry(request.userAnswers.set(EmailConfirmationQuery, emailSent))
-                    _              <- cc.sessionRepository.set(updatedAnswers)
-                  } yield {
-                    Redirect(CheckYourAnswersPage.navigate(NormalMode, request.userAnswers))
-                  }
-              }
-            case Left(ConflictFound) =>
-              auditService.audit(ReturnsAuditModel.build(
-                returnRequest, SubmissionResult.Duplicate, None, None, request
-              ))
-              Redirect(routes.YourAccountController.onPageLoad()).toFuture
-
-            case Left(e) =>
-              logger.error(s"Unexpected result on submit: ${e.toString}")
-              auditService.audit(ReturnsAuditModel.build(
-                returnRequest, SubmissionResult.Failure, None, None, request
-              ))
-              Redirect(routes.JourneyRecoveryController.onPageLoad()).toFuture
-          }
-
-        case Invalid(errors) =>
-          val errorList = errors.toChain.toList
-          val errorMessages = errorList.map(_.errorMessage).mkString("\n")
-          logger.error(s"Unable to create a VAT return request from user answers: $errorMessages")
-
-          Redirect(routes.JourneyRecoveryController.onPageLoad()).toFuture
+      if (config.correctionToggle) {
+        val correctionRequest =
```

Comment:
Could we rename this variable and vatReturnRequest to be something like validatedCorrectionRequest etc... just so it reads better, I should have noticed that earlier